source/django_package/domain/import_datas/parsing_config_file.py

```diff
@@ -0,0 +1,23 @@
+import yaml
+
+
+class ReadConfigFile:
+    config_file_keys = []
+    config_file_values = []
+    data_source = []
+    data_type = []
+
+    def parse_file(self):
+        with open('csv_config_file.yaml') as f:
+            docs = yaml.load_all(f, Loader=yaml.FullLoader)
```

Comment:
Are you expecting multiple sources to be defined in one config file? If yes, then `load_all` is fine. Otherwise use `load`.
This kind of thing should be commented in docstring.